sklearn/linear_model/_base.py

```diff
@@ -414,6 +418,21 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
         Independent term in the linear model. Set to 0.0 if
         `fit_intercept = False`.
 
+    See Also
+    --------
+    sklearn.linear_model.Ridge : Ridge regression addresses some of the
+    problems of Ordinary Least Squares by imposing a penalty on the size of
```

Comment:
This line and all the second lines of "See Also" descriptions need to be indented by 4 characters I think.